### PR TITLE
BXMSDOC-8586-blue-reinstate Reinstate Operator Content

### PR DESCRIPTION
**Pull request by emmurphy1**

### CODE REVIEW

File: doc-content/enterprise-only/openshift/operator-deploy-basic-proc.adoc [hunk 1]
```diff
@@ -17,8 +17,7 @@ ifdef::PAM[]
 +
 [IMPORTANT]
 ====
-In {PRODUCT} {ENTERPRISE_VERSION}, high-availability {CENTRAL} functionality deployment using the operator is for Technology Preview only. For more information about Red Hat Technology Preview features, see https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Support Scope]. For a fully supported high-availability deployment, use the high-availability authoring template on {OPENSHIFT} version 3.11. For instructions about deploying this template, see xref:assembly-openshift-templates[].
-====
+In {PRODUCT} {ENTERPRISE_VERSION}, high-availability {CENTRAL} functionality deployment using the operator is for Technology Preview only. For more information about Red Hat Technology Preview features, see https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Support Scope].
```
Comment (spolti) [inline doc-content/enterprise-only/openshift/operator-deploy-basic-proc.adoc]: shoudn't this url be pointing to the ibm website?

Comment (emmurphy1) [reply doc-content/enterprise-only/openshift/operator-deploy-basic-proc.adoc]: I removed the link.

File: doc-content/enterprise-only/openshift/configmaps-con.adoc [hunk 2]
```diff
@@ -22,58 +22,58 @@ data:
...
 
-The {PRODUCT} operator can use the current version and the previous version of {PRODUCT} components concurrently, with ConfigMaps for each version, for example, 7.13.0 and 7.12.1.
+The {PRODUCT} operator can use the current version and the previous version of {PRODUCT} components concurrently, with ConfigMaps for each version, for example, 8.0.0 and 7.12.1.
 
-kieconfigs-7.12.1::
-This contains the `common.yaml` configuration file. For more information, see  https://github.com/kiegroup/kie-cloud-operator/blob/7.12.1-1/rhpam-config/7.12.1/common.yaml[`common.yaml`] on GitHub. You can configure the following components using this configuration file:
+kieconfigs-8.0.0::
+This contains the `common.yaml` configuration file. For more information, see  https://github.com/kiegroup/kie-cloud-operator/blob/8.0.0-1/rhpam-config/8.0.0/common.yaml[`common.yaml`] on GitHub. You can configure the following components using this configuration file:
```
Comment (csherrar) [inline doc-content/enterprise-only/openshift/configmaps-con.adoc]: [GitHub Links] - Change URLs from /blob/ to /tree/, and versions as below:

https://github.com/kiegroup/kie-cloud-operator/blob/8.0.0-1/rhpam-config/8.0.0/
to:
https://github.com/kiegroup/kie-cloud-operator/tree/release-v7.13.x-blue/rhpam-config/8.0.0/
alternatively:
https://github.com/kiegroup/kie-cloud-operator/tree/release-v7.13.x-blue/rhpam-config/{PRODUCT_VERSION_LONG}/

[Question] - {PRODUCT_VERSION} and {PRODUCT_VERSION_LONG} point to {ENTERPRISE_VERSION} and {ENTERPRISE_VERSION_LONG} respectively. Which one should we use continuing forward?

File: doc-content/enterprise-only/openshift/using-configmaps-proc.adoc [hunk 3]
```diff
@@ -19,14 +19,14 @@ $ oc get cm | grep kieconfigs
...
 ----
-kieconfigs-7.13.0               1       64m
-kieconfigs-7.13.0-dashbuilder   1       64m
-kieconfigs-7.13.0-dbs           2       64m
-kieconfigs-7.13.0-dbs-pim       3       64m
-kieconfigs-7.13.0-dbs-servers   4       64m
-kieconfigs-7.13.0-envs         10       64m
-kieconfigs-7.13.0-jms           1       64m
-kieconfigs-7.13.0-pim           1       64m
+kieconfigs-8.0.0                1       64m
```
Comment (csherrar) [inline doc-content/enterprise-only/openshift/using-configmaps-proc.adoc]: [Attributes suggestion] - Maybe change version 8.0.0 to {PRODUCT_VERSION_LONG} or {ENTERPRISE_VERSION_LONG} throughout.

**Review by csherrar — APPROVED** [review]
One correction, suggestion, and question, otherwise looks good to me!

**Pull request closed without merging**